callouts/python/extproc/l4_example/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# Copyright 2024 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Build the proto libraries from the envoy git repo using buf.
+FROM bufbuild/buf as BUF
+RUN apk add git
+COPY ./buf.gen.yaml ./buf.gen.yaml
+RUN buf -v generate https://github.com/envoyproxy/envoy.git#subdir=api --path envoy/service/network_ext_proc/v3/network_external_processor.proto --include-imports
```

Comment:
This Dockerfile looks almost identical to callouts/python/extproc/example/Dockerfile, with the exception of (maybe only?) this line.

Not saying it needs to be done in this PR, but I wonder if the same Dockerfile could be used for the L4 and L7 examples, parameterized using a Docker --build-arg argument.